.github/workflows/checks.yml

```diff
@@ -21,17 +21,17 @@ jobs:
           node-version: 20
           cache: 'pnpm'
           registry-url: 'https://registry.npmjs.org'
+      - name: Install dependencies
+        run: pnpm install --no-frozen-lockfile
       - name: Check size
-        uses: andresz1/size-limit-action@master
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
+        run: pnpm build && pnpm test:size
   storybook:
     runs-on: ubuntu-latest
     name: storybook
     steps:
       - name: Checkout the repository
         uses: actions/checkout@v4
-      - name: Install pnpm
+      - name: Install pnpm --no-frozen-lockfile
```

Comment:
Shouldn't we preserve the lock file contents to ensure ci is building exactly what was defined?

Reply:
I would say yes, but I'm trying to unblock the release. This is the first time I've seen issues like this with pnpm. I can remove it and see if it runs clean now.

Reply:
finally